recipes/sphinx-togglebutton/meta.yaml

```diff
@@ -0,0 +1,45 @@
+{% set name = "sphinx-togglebutton" %}
+{% set version = "0.2.2" %}
+
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/sphinx-togglebutton-{{ version }}.tar.gz
+  sha256: 61f1149351643de71adf323406ad753935bbdec5790c27344da1ea7baa70a867
+
+build:
+  number: 0
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+
+requirements:
+  host:
+    - pip
+    - python
+  run:
+    - docutils
+    - python
+    - setuptools
+    - sphinx
+    - wheel
```

Comment:
Does this really need wheel and setuptools at runtime?

Reply:
Nope, I removed them and built with conda-build without problems. Updating.